dev-java/openjdk/openjdk-11.0.14_p9-r1.ebuild

```diff
@@ -175,7 +181,7 @@ src_configure() {
 		--with-freetype="${XPAK_BOOTSTRAP:-system}"
 		--with-giflib="${XPAK_BOOTSTRAP:-system}"
 		--with-harfbuzz="${XPAK_BOOTSTRAP:-system}"
-		--with-jvm-features=shenandoahgc
+		--with-jvm-features=$(usex riscv zgc shenandoahgc)
```

Comment:
does not seem to be correct.

zgc is x86_64 only.
the patch adds support for shenandoah on riscv but does not touch zgc codepoints at all?

I think this line should be left as is from the looks of it.
./configure  (via hotspot.m4) already drops unsupported jvm feature on arches where it does not work, without erroring out.

can you please double check that?

Reply:
https://wiki.openjdk.java.net/display/zgc/Main zgc not only x86_64
The patch has zgc related content changes

Reply:
Using shenandoahgc will compile with errors.

Reply:
that wiki page shows that zgc is x64 only in `openjdk11` and experimental.

also in `make/autoconf/hotspot.m4`
```
  # Only enable ZGC on supported platforms
  AC_MSG_CHECKING([if zgc can be built])
  if test "x$OPENJDK_TARGET_OS" = "xlinux" && test "x$OPENJDK_TARGET_CPU" = "xx86_64"; then
    AC_MSG_RESULT([yes])
  else
    DISABLED_JVM_FEATURES="$DISABLED_JVM_FEATURES zgc"
    AC_MSG_RESULT([no, platform not supported])
  fi

```

that makes my statement correct.

also I can't find any references for `zgc` in the patch

`grep -i zgc openjdk-11.0.14-riscv.patch` returns empty result.
and looks like `./configure` step shows that zgc feature is not enabled indeed.


so looks like this this patch, assuming shenandoah errors out, the correct will be
(outside of myconf array declaration)
`! use riscv && myconf+=( --with-jvm-features=shenandoahgc )`

Reply:
Ok, I see.